.gitlab-ci.yml

```diff
@@ -92,6 +92,17 @@ cargo-audit:
   allow_failure:                   true
 
 
+check-warnings:
+  stage:                           pre-test
+  <<:                              *docker-env
+  # except:
+  #   - /^[0-9]+$/
+  script:
+    - RUSTFLAGS="-D warnings" time cargo check --release --verbose
```

Comment:
I would not run this in release.

Reply:
And does this use stable?

Reply:
this is WIP yet, experimenting how it'll be better

Reply:
Okay :)

Reply:
Yep, stable is by default in the CI